docs/howto/manage-machines.md

```diff
@@ -23,7 +23,7 @@ To add a new machine to a model, run the `add-machine` command, as below. `juju`
 juju add-machine
 ```
 
-The command also provides many options. By using them you can customize many things. For example, you can provision multiple machines, specify the Ubuntu series to be installed on them, choose to deploy on a lxd container *inside* a machine, apply various constraints to the machine (e.g., storage, spaces, ...) to override more general defaults (e.g., at the model level), etc.
+The command also provides many options. By using them you can customize many things. For example, you can provision multiple machines, specify their, choose to deploy on a LXD container *inside* a machine, apply various constraints to the machine (e.g., storage, spaces, ...) to override more general defaults (e.g., at the model level), etc.
```

Comment:
typo?

Reply:
Yeah, I meant to say "specify the desired base". Fixed now, thanks!